### PR TITLE
Clarify `sentry_on` value is not quoted with example

### PR DESCRIPTION
**Pull request by wolfier**
Clarify the value for `sentry_on` is not quoted by providing an example.

---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/main/UPDATING.md).

### CODE REVIEW

File: docs/apache-airflow/logging-monitoring/errors.rst [hunk 1]
```diff
@@ -29,13 +29,14 @@ First you must install sentry requirement:
 
    pip install 'apache-airflow[sentry]'
 
-After that, you need to enable the integration by set ``sentry_on`` option in ``[sentry]`` section to ``"True"``.
+After that, you need to enable the integration by set ``sentry_on`` option in ``[sentry]`` section to ``'True'``.
```
Comment (kaxil) [inline docs/apache-airflow/logging-monitoring/errors.rst]:
```suggestion
After that, you need to enable the integration by set ``sentry_on`` option in ``[sentry]`` section to ``True``.
```

Comment (wolfier) [reply docs/apache-airflow/logging-monitoring/errors.rst]: Should I remove the single quotes for `sentry_dsn` as well?

Comment (uranusjr) [reply docs/apache-airflow/logging-monitoring/errors.rst]: +1 to removing quotes to both. These are INI, not Python, the quotes just shouldn’t exist in the first place.

Comment (kaxil) [reply docs/apache-airflow/logging-monitoring/errors.rst]: yes please remove the quotes

**Review by uranusjr — APPROVED** [review]

**Review by kaxil — APPROVED** [review]

**Pull request merged**